CHANGELOG.md

```diff
@@ -58,6 +58,7 @@
 * Unable to save new references and ackowledgments on flaws (`OSIDB-2206`)
 * Reset affects when flaw is reset (`OSIDB-2793`)
 * New public comments are not displayed (`OSIDB-2700`)
+* `updated_dt` field should be updated on flaw save (`OSIDB-2933`)
```

Comment:
We aim to have changelog as user friendly as possible.
For example for this case I'd add something like:
```suggestion
* Fix 409 errors on saving affects  (`OSIDB-2933`)
```

Reply:
Ah yeah, good point, or even 'Fix outdated model (409) errors on saving affects'

Reply:
> We aim to have changelog as user friendly as possible.
> For example for this case I'd add something like:
> ```suggestion
> * Fix 409 errors on saving affects  (`OSIDB-2933`)
> ```

Yeah that's a bit more user-friendly 😂
Will change the message and merge tomorrow morning